blockchain/blockchain.go

```diff
@@ -1139,6 +1137,10 @@ func (bc *blockchain) commitBlock(blk *block.Block) error {
 			return errors.Wrapf(err, "failed to put smart contract receipts into DB on height %d", blk.Height())
 		}
 	}
+
+	// emit block to all block subscribers
+	bc.emitToSubscribers(blk)
```

Comment:
why is it necessary to move code here?

Reply:
my bad, it's not necessary. I originally think we modify receipt in sf.Commit()